vignettes/protein_structure_workflow.Rmd

```diff
@@ -0,0 +1,399 @@
+knitr::include_graphics("figures/interaction_2hwg.png")
+```
+
+Mapping of significantly changing peptides onto the pgk structure reveals that most of the protein seems to be structurally affected by the treatment with 3-phosphoglyceric acid (3PG). Based on UniProt annotations and the similarity of the protein to its homologous in other organisms, the 3PG binding site is located on the left site indicated with a black arrow. All residues involved in the interaction with 3PG are not part of a significantly changing peptide. Due to the compact size of the protein and the even distribution of significant pepties it is hard to draw any conclusions about potential structural effects of the treatment.
+
+```{r peptide_map_1zmr, eval = test_protti, echo = FALSE, fig.align = "center", fig.cap = "**pgk (1ZMR) with mapped significantly changing peptides.**"}
+knitr::include_graphics("figures/peptide_map_1zmr.png")
+```
+## Mapping of amino acid scores
+
+In order to narrow down and prioritize structurally affected protein regions we have developed the amino acid scoring function `calculate_aa_scores()`. For each amino acid position in the protein the function calculates the average product of the -log10(adjusted p-value) and the absolute log2(fold change) per peptide covering the amino acid position. In other words the information from all peptides in a specific region is integrated, which strengthens the confidence in a structural change being true due to it averaging out false positive peptides. One downside of this approach is that it relies on very good coverage of the protein since otherwise false positive peptides solely covering a specific region are not distinguishable from real hits and they can recieve very high scores. In our example we are lucky that both proteins have a very high coverage.
+
+The `map_peptides_on_structure()` function is capable of mapping a continuous value to a structure. In this case that values are again scaled between 50 and 100. In this case it is possible that values fall in between 50 and 100 which are displayed as a colour gradient step in between the colour in the middle of the gradient (50) and the colour at the end of the gradient (100). 
+
+_Note: If multiple structures are mapped at the same time, by default each structure scales its map values between 50 and 100 based on its minimum and maximum score. It is however possible to use one scale for all structures that are mapped with the `scale_per_structure` argument of the `map_peptides_on_structure()` function. In this case only one of the structures (the one containing the highest score) will have the 100 value and all other structures a value lower than that as their maximum value depending on their highest score. This means that the scale of colours between both structures is comparable._
```

Comment:
delete "however" in second sentence

Reply:
Maybe you should add the "scale_per_structure" argument in your example. I don't think this explanation is very clear